docs/INSTALLING.md

```diff
@@ -96,6 +96,10 @@ Point your browser (or make a direct request) to `http://api.localhost:49235/use
 }
 ```
 
+If your browser does not support localhost loopback (address not found error), please use `http://api.lvh.me:49235/users` address instead (lvh.me resolves itself and all subdomains to your `localhost` accordingly).
+
+Note: some browsers, e.g. Safari, Firefox, can ask to download file instead of displaying its contents directly. This is [known issue for JSON API](https://github.com/json-api/json-api/issues/1048) and if you find this inconvenient, please use Chrome or Opera.
```

Comment:
", e.g. Safari, Firefox, can" => "like Safari and Firefox may"

"This is a"

Also maybe add recommendation to use tools like https://www.getpostman.com/ or https://paw.cloud/